packages/import-manager/yarn.lock

```diff
@@ -0,0 +1,1289 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We don't want `yarn.lock` files in the individual package directories.. just the single `yarn.lock` at the top of the tree. Running `yarn install` at the top seems to preserve this, but running it in a subdirectory does not. As a testing step, it's probably useful to run `yarn install` from a subdirectory (that simulates what someone will do when the `npm install` a single package), and maybe we should do that from CI for all packages, but I don't think we should be committing any files generated as a side-effect.

Reply:
Good point. I think what happened is that I didn't add them as workspace package until late in the game, so they weren't making use of the workspace as they should (for instance, they couldn't find the workspace eslint.) Next time I'll make sure to list them as workspace packages first.